### ISSUE TITLE
[Suggestion] Add "Clear Selection" Button to Area Editor (simple mode) menu.

### ISSUE THREAD

**Issue by Eiion** [maruohon/litematica#346]
**Is your feature request related to a problem? Please describe.**
The only way to get rid of a selection using the simple mode Area Selection is by going to the menu and manually remove all entries from the Area Editor coordinate input boxes.

**Describe the solution you'd like**
A simple button that would clear any entry from the Area Editor menu which allows us to click it to get rid of the selection instead of having to delete every entry by hand.

**Comment by maruohon** [comment]
It's actually not possible to remove the selection box at all in the Simple mode (by design). I think if you are removing/clearing the coordinate inputs, it's just reverting them all to 0, 0, 0? There is an option to just disable the area selection rendering, and I could add that as a button to the Area Editor menu as well.

**Comment by Eiion** [comment]
No, that's not what's happening. Clearing the coordinate o not removes it - just as I've described it in the original post.
And even if that is what is happening code could be corrected to avoid that from happening. That would be the clean way to do it. Everything else is only ways around a clean solution.
